test_cases/test_persons.py

```diff
@@ -1,17 +1,17 @@
 import unittest
-from app.person import Staff, Fellow
+from ..app.person import Staff, Fellow
```

Comment:
Always leave a space between standard library imports and your third party or local code imports. 

> Imports should be grouped in the following order:
> 
> standard library imports
> related third party imports
> local application/library specific imports
> You should put a blank line between each group of imports.

Ref: https://www.python.org/dev/peps/pep-0008/#imports